src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -91,6 +91,10 @@ public boolean isResultZipFound(String chainTaskId) {
         return new File(getResultZipFilePath(chainTaskId)).exists();
     }
 
+    public String getResultProxyUrl(String chainTaskId) {
+        return iexecHubService.getTaskDescription(chainTaskId).getResultStorageProxy();
+    }
```

Comment:
With other comments, this method could be avoided by injected URL from calling methods